lib/maintainers.nix

```diff
@@ -18,7 +18,7 @@
     github = "Bodleum";
     githubId = 60107449;
     name = "Daniel Laing";
-    keys = [ { fingerprint = "0821 8B96 DC73 85E5 BB7C  A535 D264 3BD2 13BC 0FA8"; } ];
+    keys = [{ fingerprint = "0821 8B96 DC73 85E5 BB7C  A535 D264 3BD2 13BC 0FA8"; }];
```

Comment:
It seems you're using a different formatter to us. Could you instead try using `nix fmt` to format using the formatter we've configured in our flake?

Reply:
Also, if you enter a devshell (e.g. using direnv, or by running `nix develop`), a git "pre-commit" hook will be installed that checks the formatting is correct each time you make a commit. :grin: